packages/create-svelte/bin.js

```diff
@@ -77,7 +77,7 @@ const options = await p.group(
 
 		features: () =>
 			p.multiselect({
-				message: 'Select additional options',
+				message: 'Select additional options (use arrow keys to navigate and SPACE to toggle options)',
```

Comment:
Maybe make it a little shorter, so that it's less likely to break onto two lines?

```suggestion
				message: 'Select additional options (use arrow keys/space bar)',
```

Reply:
Sounds good to me 👍